### PR TITLE
[ADD] partner_contact_birthplace

### PR DESCRIPTION
**Pull request by SimoRubi**
Adding birth city, state and country to the partner

### CODE REVIEW

**Review by hurrinico — APPROVED** [review]
👍

**Review by eLBati — APPROVED** [review]

**Review by tafaRU — APPROVED** [review]

File: partner_contact_birthplace/views/res_partner.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!-- Copyright 2018 Simone Rubino - Agile Business Group
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="view_personal_information_birthplace" model="ir.ui.view">
+        <field name="name">Birthplace field</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_contact_personal_information_page.personal_information"/>
+        <field name="arch" type="xml">
+            <data>
```
Comment (tarteo) [inline partner_contact_birthplace/views/res_partner.xml]: `<data>` is not necessary.

**Review by tarteo — APPROVED** [review]
No blocking comments :+1:

File: partner_contact_birthplace/views/res_partner.xml (hunk 1 above)
Comment (SimoRubi) [reply partner_contact_birthplace/views/res_partner.xml]: fixed @tarteo

**Comment by tafaRU** [comment]
@SimoRubi, please squash commits to only one.
After that I'll merge it considering that LINT errors are no related to your module.

**Comment by SimoRubi** [comment]
done @tafaRU

**Pull request merged**